tests/unit/client.py

```diff
@@ -335,35 +301,37 @@ def test_nosuch_node_add_nic(self):
             C.node.add_nic('abcd', 'eth0', 'aa:bb:cc:dd:ee:ff')
 
     def test_remove_nic(self):
-        result = C.node.remove_nic('node-08', 'eth0')
-        assert result is None
+        assert C.node.remove_nic('node-08', 'eth0') is None
 
     def test_remove_duplicate_nic(self):
         C.node.remove_nic('node-08', 'eth0')
         with pytest.raises(Exception):
             C.node.remove_nic('node-08', 'eth0')
 
     def test_node_connect_network(self):
-        result = C.node.connect_network(
+        assert C.node.connect_network(
                 'node-01', 'eth0', 'net-01', 'vlan/native'
-                )
-        assert result is None
+                ) is None
 
     def test_node_start_console(self):
-        result = C.node.start_console('node-01')
-        assert result is None
+        assert C.node.start_console('node-01') is None
 
     def test_node_stop_console(self):
-        result = C.node.stop_console('node-01')
-        assert result is None
+        assert C.node.stop_console('node-01') is None
 
 
 # FIXME: I spent some time on this test. Looks like the pytest
 # framework kills the network server before it can detach network.
-# def test_node_detach_network(self):
-# C.node.connect_network('node-04', 'eth0', 'net-04', 'vlan/native')
-# result = C.node.detach_network('node-04', 'eth0', 'net-04')
-# assert result is None
+# Explanation: The test is unreliable because of following reasons.
+# When using a real switch driver for network, haas network server
+# times out waiting for response from switch while using mock switch
+# driver, the networking_action queue takes longer to execute the request
+# the testing server kills the servers when other tests are completed.
```

Comment:
This paragraph scans very poorly. I read it 7 or 8 times and still wasn't entirely sure what it meant. Then I went back and looked at your comment, which is more understandable.

You're missing some punctuation, but that's not everything that makes it hard to understand.

The explanation you gave in a comment